addons/event_booth_sale/models/product.py

```diff
@@ -21,6 +21,14 @@ def _detailed_type_mapping(self):
         type_mapping['event_booth'] = 'service'
         return type_mapping
 
+    @api.model
+    def _get_product_types_possible_zero_price_sale(self):
+        """
+        Returns a list of detailed types (`product.template.detailed_type`) that can ignore the
+        `prevent_zero_price_sale` rule when buying products on a website.
+        """
+        return super()._get_product_types_possible_zero_price_sale() + ["event_booth"]
```

Comment:
If it's only when we have a website then it would be better in `website_event_booth_sale`